Frameworks/MQTTnet.NetStandard/Client/MqttClient.cs

```diff
@@ -194,6 +194,47 @@ public MqttClient(IMqttClientAdapterFactory channelFactory, IMqttNetLogger logge
             }
         }
 
+        public async Task PublishAsync(MqttApplicationMessage applicationMessage)
```

Comment:
Sorry but I don't agree here. A whole copy of this method including all the new interface methods only for avoiding an array with one element? Or is there a different reason?

Reply:
I guess the impact is minimal. The main idea was that most calls are single messages, and yet we were enumerating a collection at every call just to avoid having a specialized method and a collection one. The params only pattern is not very common on most libraries and even on dotnet itself, apart from places like string.format